cmd/res/configuration.toml

```diff
@@ -14,16 +14,27 @@
 #################################################################################
 
 # This is a TOML config file for edgex-mongo init service.
+# If the application works in secure enabled environment, [SecretStore] section is meaningful, meanwhile [Credentials] section is ignored
+# and vice versa
+
 [Writable]
-    LogLevel = "INFO"
+  LogLevel = "INFO"
 
 [Service]
-    BootTimeout = 30000
+  BootTimeout = 30000
+
+[SecretStore]
+  Server = "localhost"
+  Port = 8200
+  DBStem = "/v1/secret/edgex/dbcredentials"
+  CACertPath = "/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem"
+  TokenPath = "/vault/config/assets/resp-init.json"
+  ServerName = "edgex-vault"
```

Comment:
what's the ServerName for? is this the SNI setting for https? if so this won't work it will need to be localhost to match the `Server` key

Reply:
Yes,  this is the Server Name Identifier. I used to use SNI as a variable name, but I have renamed it. But it looks it is confusing now, I will rename it back
The value is fixed